changelogs/fragments/446-system-documentation-links-fix.yaml

```diff
@@ -0,0 +1,3 @@
+---
+bugfixes:
```

Comment:
Since this PR contains only documentation related changes, the category could be changed to 'trivial'

Reply:
Done.

I debated on whether or not this qualified as a bug fix because it is fixing "broken" links, But since these changes have no impact on functionality, I agree with your suggestion. I and have changed the PR type to "Docs Pull Request", have added the "documentation" label to it (which means it no longer requires a fragment file to pass the workflow changelog check), and have changed the fragment designation to 'trivial'.